unit_tests/test_api.py

```diff
@@ -304,3 +304,57 @@ def test_reset_update_center(self):
             url=url)
         self.fakes.jenkins.scripts[script] = ""
         self.assertIsNone(self.api.set_update_center())
+
+    def test_check_update_center(self):
+        """
+        The set_update_center() method runs a groovy script to modify the
+        update center url to default when no url value is given.
+        """
+        url = "https://updates.jenkins.io/stable/update-center.json"
+        script = SET_UPDATE_CENTER_SCRIPT.format(
+            url=url)
+        self.fakes.jenkins.scripts[script] = ""
+        self.assertIsNone(self.api.set_update_center())
+
+    def test_check_update_center(self):
```

Comment:
This has the same name as the previous test.

Reply:
Removed duplicate in 733c0da